requirements.txt

```diff
@@ -1,8 +1,8 @@
 # we require torch nightly for flash attention support
 # NOTE: you might need to replace "cu118" with "cpu" or "cu117"
---extra-index-url https://download.pytorch.org/whl/nightly/cu118 --pre
+--extra-index-url https://download.pytorch.org/whl/nightly/cpu --pre
```

Comment:
If you prefer installing `cpu` by default, the note above needs to be updated and also the same code in `setup.py`

I chose cuda by default because I would expect that most users who try the repo have a cuda device.

Reply:
Sorry my bad, this wasn't intentional

Reply:
Fixed